ocs_ci/ocs/cluster.py

```diff
@@ -2706,3 +2739,29 @@ def get_lvm_full_version():
     image = getattr(redhat_operators_catalogesource_ocs, "data")["spec"]["image"]
     full_version = image.split(":")[1]
     return full_version
+
+
+def get_lvm_pods_names():
+    """
+    Get lvm pods name (without operator)
+    Returns:
+        list(POD): list of pods objects.
+    """
+    lvm_pods = get_pods_having_label(
+        label="app.kubernetes.io/managed-by=lvm-operator",
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+    )
+    return lvm_pods
+
+
+def wait_for_lvm_pod_running():
+    """
+    Wait for all lvm pods to be running.
+    """
+
+    pod_name_list = []
+    for lvm_pod in get_lvm_pods_names():
+        pod_name_list.append(lvm_pod["metadata"]["name"])
+    wait_for_pods_to_be_running(
+        pod_names=pod_name_list, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+    )
```

Comment:
these 2 functions are already in https://github.com/red-hat-storage/ocs-ci/pull/6258/files#diff-a4d73d82c94d13634aded838ea2490c117996218f405a8da15f9ecea093f3f6dR2620-R2641

Reply:
I need to access this data without initializing the class because the class will fail if LVM pods are not up like after reboot I can't initialize lvm unless I know that the pods are up.

Reply:
And in the class init I'm checking also for the pods to be running.